tests/test-reduce.py

```diff
@@ -551,3 +580,18 @@ def test_corr_multiple():
     assert_equals(D1, dt.Frame([[1.0], [a], [b], [-b]]))
     assert_equals(D2, dt.Frame([[-b], [-c], [-1.0], [1.0]]))
     assert_equals(D3, dt.Frame([[1.0], [1.0], [1.0], [1.0]]))
+
+
+
+#-------------------------------------------------------------------------------
+#  countna
+#-------------------------------------------------------------------------------
+
+
+@pytest.mark.parametrize("src", srcs_all)
+def test_dt_count_na(src):
+    dt0 = dt.Frame(src)
+    ans = dt0.countna()
+    dtr = dt0[:, dt.countna(f[:])]
+    frame_integrity_check(dtr)
+    assert_equals(ans, dtr)
```

Comment:
Can you please move this test into a file dedicated to `countna()` function? It should be "tests/dt/test-countna.py".
We're trying to [move to a system](https://github.com/h2oai/datatable/issues/3030) where each function has a dedicated test file, linked from the corresponding `.rst` file. This will let us see clearer which functions are well-tested, and which aren't.

Reply:
Ack